dev/custom-pages/homepage.scss

```diff
@@ -896,13 +916,22 @@ $color-linkedin: #0077b5;
   }
 
   .callout-events {
+    background-position: 100% 0;
     background-repeat: no-repeat;
     background-size: contain;
-    background-position: 100% 0;
     overflow-x: hidden;
+    padding: 5px 0 75px 0;
 
     @media (max-width: $screen-sm-max) {
-      background-image: none !important;
+      background: $black url($events-background-mobile-url) !important;
+      background-position: 50% 75% !important;
+      background-repeat: no-repeat !important;
+      background-size: cover !important;
+      padding: 30px 0 300px;
+    }
+
+    @media (max-width: $screen-xs-max) {
+      padding: 5px 0 220px;
```

Comment:
You may want to add a separate breakpoint that increases padding-bottom for ~480px and lower; with 220px bottom padding I'm seeing some text overflow into the background in QA.

The mobile background image also has a lot of plain black empty space that could be cropped out--if the image were cropped without the empty space, you could do something like:
```
background-position: center bottom;
padding-bottom: <some value in vw units that corresponds to the aspect ratio of the image>
```
to ensure the padding always matches the image's dimensions.